mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,16 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.eraseOrder = async (req, res) => {
+  Order.findByIdAndDelete(req.params.orderId)
+    .then(deletedOrder => {
+      if (!deletedOrder) {
+        return res.status(404).json({ error: "No such order found" });
+      }
+      res.json({ success: "Order has been deleted" });
+    })
+    .catch(error => {
+      res.status(500).json({ error: errorHandler(error) });
```

Comment:
The error response object should have a consistent structure throughout the application. In this case, the error response object should include a message and error field, similar to other error responses in the file. It should be changed to `res.status(500).json({ error: errorHandler(error), message: 'Internal Server Error' });` to maintain consistency.